src/components/DropdownV2/DropdownV2-story.js

```diff
@@ -88,7 +116,8 @@ storiesOf('DropdownV2', module)
       `,
     })(() => (
       <div style={{ width: 300 }}>
-        <DropdownSkeleton />&nbsp;
+        <DropdownSkeleton />
+        &nbsp;
```

Comment:
This looks like a side effect of prettier. The commit hooks also changed the bullets in .md files and added various newlines to .js files.

Reply:
Hi @justinpurcell1! Most likely this is due to using something other than `yarn` to manage dependencies (e.g. prettier version is different locally than what the project uses).